src/julia/Integer.jl

```diff
@@ -58,10 +58,14 @@ function expressify(a::Integer; context = nothing)
     return a
 end
 
-function show(io::IO, R::Integers)
+function show(io::IO, R::Integers{BigInt})
    print(io, "Integers")
 end
 
+function show(io::IO, R::Integers{T}) where T
+   print(io, "Integers{$T}()")
+end
```

Comment:
can you add a similar show change for `Rationals` and `Floats` as well? I think it would be great to have these three as aligned as possible

Reply:
I have removed the printing tweak from this PR and put it into a separate PR. It can adjusted there (or not) as people have time, without holding up this useful PR (including the bug fix in it) further